config/org.go

```diff
@@ -42,6 +47,18 @@ func (o *Orgs) Contains(orgName string) bool {
 	return false
 }
 
+func (o *Orgs) GetOrgListFilenameAndPath(configDir string) string {
```

Comment:
I'm not a fan of this - a lot of the work I did recently was to make it so that nothing outside of the `config` package has to know or care that the config comes from a file on disk or even that it's yaml.

This seems to add knowledge about config files to the public API, breaking the abstraction of the `config` package.